### PR TITLE
Update printer & SVS setup guide for Tails 4

### PR DESCRIPTION
**Pull request by eloquence**
Towards #4893.

Printer guide:
- Printer settings are in a different location in settings
- Updated screenshots

SVS setup:
- Date & time are in a different location in settings
- It is no longer necessary to disable automatic decompression as
  it is disabled by default in Tails 4

## Status

Ready for review

## Checklint

- [x] Linter is cool with this change after being bribed with salted caramels

### CODE REVIEW

File: docs/set_up_svs.rst [hunk 1]
```diff
@@ -42,14 +42,6 @@ click **Add**, then click **Start Tails**.
...
 
-Ensure Filenames are Preserved
-------------------------------
-
-In order to preserve filenames when you decrypt submissions, on each *Secure
-Viewing Station*, you should open a **Terminal** and type the following commands:
-
-.. include:: includes/tails-svs-nautilus.txt
-
 Correct the System Time
```
Comment (zenmonkeykstop) [inline docs/set_up_svs.rst]: Screenshot in this section is Tails3-specific. Could be updated.

File: docs/tails_printing_guide.rst [hunk 2]
```diff
@@ -36,59 +36,52 @@ increase the chances for a seamless installation experience in Tails.
...
-
 Printing from the Command Line
 ------------------------------
 
-You can also easily print from the command line using the ``lp`` command; if
-you haven't already set your installed printer as default in the GUI, you can
-quickly do so by adding this line to your ``~/.bashrc`` file, or entering this
-directly into the terminal:
+After you have configured your printer, you can also easily print from the
+command line using the ``lp`` command; if you haven't already set your installed
```
Comment (zenmonkeykstop) [inline docs/tails_printing_guide.rst]: Could this be broken up into 2 sentences rather than two disjoint clauses separated by ';'?

Comment (eloquence) [reply docs/tails_printing_guide.rst]: Sure thing, done.

File: docs/set_up_svs.rst (hunk 1 above)
Comment (eloquence) [reply docs/set_up_svs.rst]: Thanks for taking care of that one, LGTM

**Comment by conorsch** [comment]
Changes LGTM. I've approved, and will dismiss @zenmonkeykstop's review based on @eloquence's confirmation of changes.

**Pull request merged**